ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -118,7 +118,7 @@ class TransactionDetailsScreen extends React.Component<Props, State> {
     if (
       whereAmIFrom(this.props.nav).fold(
         false,
-        r => r === "WALLET_HOME" || r === "WALLET_PAYMENT_METHOD_DETAIL"
+        r => r === "WALLET_HOME" || r === "WALLET_CREDIT_CARD_DETAIL"
```

Comment:
```suggestion
        r => r === ROUTES.WALLET_HOME || r === ROUTES.WALLET_CREDIT_CARD_DETAIL
```
Could it be an improvement to have a compile time error?

Reply:
Better for sure, changed in e02d46a9e681ac7a16777368f766837e18776f77